bibliopixel/layout/strip.py

```diff
@@ -23,17 +23,29 @@ def __init__(self, drivers, threadedUpdate=False,
             self.set = self._setScaled
             self.numLEDs = self.numLEDs / self.pixelWidth
 
+        self.coordMap = coordMap
+        if self.coordMap:
+            if len(self.coordMap) != self.numLEDs:
+                raise ValueError('coordMap length must equal total number of pixels!')
+            self.set_base = self._set_strip_mapped
+        else:
+            self.set_base = self._set_base
+
         self.set_pixel_positions(gen_strip(self.numLEDs))
 
+    def _set_strip_mapped(self, pixel, color):
+        pixel = self.coordMap[pixel]
```

Comment:
The reason I do this is so that if there's no coordMap at all then there's no if statement theoretically saving CPU cycles. I do it other places... like with scaled pixels.
You are right about `pixel` but it's already that way in the docs... I'd rather change the place that uses pixel to mean color.